### ISSUE TITLE
Downloads are no longer queued

### ISSUE THREAD

**Issue by ehfraim** [downthemall/downthemall#424]
**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser and version: Firefox 108b6
 - DownThemAll! version: 4.5.2 (latest)

**Describe the bug**
After adding one or several links from Save link with DownThemAll! download window, no downloads are queued in the download manager. Downloading the file with Firefox native downloading works, so the file is present.
Adding downloads with DownThemAll onelick also does nothing.
In both cases, no manager window is opened after pressing the Download-button.

**To Reproduce**
1. Go to any website, eg https://github.com/downthemall/downthemall/releases
2. Right click a file link and click Save link with DownThemAll or Save link with Oneclick
3. Or open DownThemAll manager and press the Add download button, paste the file url in the download field
4. Press download
5. There is no error, it just silently fails.

**Expected behavior**
The files should be added to the manager and downloaded.

**Additional context**
It worked like it should very recently.

**Comment by nmaier** [comment]
hmm. I cannot reproduce, and there was actually no update recently,

Can you check this out please? https://github.com/downthemall/downthemall/wiki/Debug-log

**Comment by ehfraim** [comment]
An error occurs when I click "download" in the download window, this is the full message
```
Uncaught (in promise) TypeError: can't access property "getAll", this.db is undefined
    getAll db.ts:241
    init man.ts:115
    getManager man.ts:519
    queue api.ts:73
db.ts:241:4
    singleTurbo api.ts:143
    AsyncFunctionThrow self-hosted:811
    (Async: async)
    findSingleItem background.ts:406
    AsyncFunctionNext self-hosted:807
    (Async: async)
    onClickedDTATurboLink background.ts:503
    onClicked background.ts:419
    onClicked self-hosted:1356
    apply self-hosted:2736
    applySafeWithoutClone ExtensionCommon.jsm:625
    applySafe ExtensionCommon.jsm:608
    sync ExtensionCommon.jsm:2514
    listener ext-menus.js:280
    withHandlingUserInput ExtensionCommon.jsm:116
    listener ext-menus.js:279
    apply self-hosted:2736
    applySafeWithoutClone ExtensionCommon.jsm:625
    fire ExtensionChild.jsm:832
    recvRunListener ExtensionChild.jsm:836
    recvRunListener self-hosted:1356
    _recv ConduitsChild.jsm:82
    receiveMessage ConduitsChild.jsm:188

```
I didn't mention it before, but I have two different firefox installations, one on the beta channel and one developer channel, they use two different profiles and both have the same problem.

**Comment by ehfraim** [comment]
In the debug console there is a error already present before doing anything, I don't know if that is usually there or if it's relevant:
```
Failed to init components Error: Could not find closed window using sessionId undefined. <empty string> Error: Could not find closed window using sessionId undefined. background.ts:677:12
    ts background.ts:677
```

I tried making a new profile and adding downthemall, and it works fine. I went back to my old profile and uninstalled downthemall, restarted Firefox and reinstalled downthemall, and It works again. I also tried rebooting my computer and it still works. Some other person on reddit had the same problem but it stopped working for them after a reboot, that's why I tried it. 
https://old.reddit.com/r/firefox/comments/xkuxzl/downthemall_broken_in_105/

Firefox must have done something to both my profiles, I guess we can close this issue unless you want to dig into why the database went missing?
